ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -1322,6 +1329,14 @@ impl fmt::Display for DisplayableInst<'_> {
             Inst::FCmp { lhs, pred, rhs, .. } => {
                 write!(f, "{pred} {}, {}", lhs.display(self.m), rhs.display(self.m))
             }
+            Inst::Promote { val, safepoint, .. } => {
+                write!(
+                    f,
+                    "promote({}) {}",
```

Comment:
I'd say remove the `()` around the promoted operand, so this instruction format like the other AOT instructions?

Reply:
Sure, fixed here (ba681044430dafa8ef109f085623839c77043211)